docs/sources/_index.md

```diff
@@ -3,20 +3,52 @@ cascade:
   FULL_PRODUCT_NAME: Grafana Explore Profiles
   PRODUCT_NAME: Explore Profiles
 description: Learn how to use Explore Profiles to understand and troubleshoot
-  your application and services.
+  your applications and services.
 keywords:
   - Explore Profiles
   - Profiles
 title: Explore Profiles
 menuTitle: Explore Profiles
 weight: 100
+hero:
+  title: Explore Profiles
+  level: 1
+  width: 100
+  height: 100
+  description: Use Explore Profiles to investigate and identify issues using profiling data.
+cards:
+  title_class: pt-0 lh-1
+  items:
+    - title: Concepts
+      href: ./concepts/
+      description: Learn the concepts you need to understand to use profiling.
+      height: 24
+    - title: Get started
+      href: ./get-started/
+      description: How do you use profiling data to investigate an issue? Start here.
+      height: 24
+    - title: Determine your use case
+      href: ./troubleshooting/
+      description: Chones one of the wwo most common use cases to guide your exploration.
```

Comment:
"Choose"? Also, do you mean "one of the two"?

Reply:
Yes :)  ERF. I should have run spell check... 
```suggestion
      description: Choose one of the two most common use cases to guide your exploration.
```